### DISCUSSION TITLE
Map coordinates & paths

### DISCUSSION THREAD

**Discussion by ero-qt** [Leaflet/Leaflet#8799] [Q&A]
Hi, two questions;

I'm using `L.CRS.Simple` for a map of a video game. This messes with a bunch of coordinate-related things. Obviously, the `LatLngBounds` don't correspond to the real pixel positions in the original image. Additionally, I seem to have to set `Map.maxBounds` and `TileLayer.bounds` to `[[0, 0], [-tileSize, tileSize]]` for the bounds to be correct. This is pretty annoying, especially when it comes to placing markers; they have to be within that `[tileSize, tileSize]` range as well, which is usually just `256 x 256`.

Is there a way for me to change these bounds or coordinates somehow? I would like to use the image's pixel size, or at least set it to something more sensible, like `1000 x 1000`.

---

Does Leaflet have the ability to display "paths"? A path would consist of two or more nodes that are connected by a smoothly curving line. If not, is there some work around I can use to achieve something similar?

**Comment by IvanSanchez** [comment]
Use the game's coordinate system as your own coordinate system, instead of basing everything on pixels. Make your images/polygons/etc fit the coordinate system instead of the other way around.

---

> Does Leaflet have the ability to display "paths"?

See the plugins list.

**Reply by ero-qt** [reply]
> Use the game's coordinate system as your own coordinate system, instead of basing everything on pixels. Make your images/polygons/etc fit the coordinate system instead of the other way around.

This isn't really feasible. It's a Unity game, the coordinates are
1. really difficult to obtain, and
2. just not really representative of anything ever.

It would be much easier to simply use the image's pixels to base the position of markers off of; anyone can obtain those positions and it'd be the easiest thing in the world to place things from there.

That said, how would I begin doing this?

**Reply by ero-qt** [reply]
@IvanSanchez Please enlighten me when you get a chance.

**Comment by ero-qt** [comment]
I'm bumping this in hopes that someone knows an answer to my problem here. What Ivan suggested above (using the game's coordinate system) is an option that I may consider, but I think it comes with its own issues. I don't really know how to set the tiles' sizes and the corresponding coordinates separately. Additionally, if I were to take this approach, I would have to set the `0, 0` point arbitrarily, since it's unlikely to be perfectly centered.
